gnome/gstreamer1-gst-plugins-good/Portfile

```diff
@@ -26,146 +27,131 @@ master_sites        https://gstreamer.freedesktop.org/src/${my_name}/
 }
 
 variant pulseaudio description {Enable PulseAudio plugin} {
     depends_lib-append      port:pulseaudio
-    configure.args-replace  --disable-pulse \
-                            --enable-pulse
+    configure.args-append   -Dpulse=enabled
 }
 
 variant x11 {
     depends_lib-append      port:xorg-libX11 \
                             port:xorg-libXfixes \
                             port:xorg-libXdamage \
                             port:xorg-libXext
-    configure.args-append   --with-default-videosink=ximagesink
-    configure.args-replace  --disable-x \
-                            --enable-x
+    configure.args-append   -Dximagesrc=enabled \
+                            -Dximagesrc-xshm=enabled \
+                            -Dximagesrc-xdamage=enabled \
+                            -Dximagesrc-xfixes=enabled
 }
 
-default_variants    +pulseaudio +x11 +gtk3 +qt
-
-variant qt description {Enable QT5 MAC windowing elements } {
+# Requires gstreamer-gl
+variant qt description {Enable Qt5 MAC windowing elements } {
     PortGroup               qt5 1.0
 
     qt5.depends_component   qtdeclarative \
                             qtmacextras
 
-    configure.args-replace  --disable-qt \
-                            --enable-qt
+    configure.args-replace  -Dqt5=disabled \
+                            -Dqt5=enabled
 
     # avoid:
-    #    ${prefix}/libexec/qt5/include/QtCore/qcompilerdetection.h:562:6: error: Qt requires a C++11 compiler and yours does not seem to be that.
+    # ${prefix}/libexec/qt5/include/QtCore/qcompilerdetection.h:562:6: error: Qt requires a C++11 compiler and yours does not seem to be that.
     compiler.cxx_standard   2011
     configure.cxxflags-append -std=c++11
 }
 
-if {${universal_possible} && [variant_isset universal]} {
-    set merger_host(x86_64) x86_64-apple-${os.platform}${os.major}
-    set merger_host(i386) i686-apple-${os.platform}${os.major}
-    set merger_configure_args(x86_64) --build=x86_64-apple-${os.platform}${os.major}
-    set merger_configure_args(i386) --build=i686-apple-${os.platform}${os.major}
-} else {
-    if {${build_arch} eq "i386"} {
-        configure.args-append \
-            --host=i686-apple-${os.platform}${os.major} \
-            --build=i686-apple-${os.platform}${os.major}
-    } elseif {${build_arch} eq "x86_64"} {
-        configure.args-append \
-            --host=${build_arch}-apple-${os.platform}${os.major} \
-            --build=${build_arch}-apple-${os.platform}${os.major}
-    }
-}
-
-# SSE register return with SSE disabled
-# <rdar://problem/9694837> - Fixed in Xcode 4.2 (clang 211.10.1)
-# blacklist compilers that don't support C11 typedef redefinition
-# gstflvmux.h:73: error: redefinition of typedef ‘GstFlvMuxPadClass’
-compiler.c_standard 2011
+default_variants            +pulseaudio +x11 +gtk3
 
 livecheck.type      regex
 livecheck.url       ${master_sites}
 livecheck.regex     "${my_name}-(\\d+\\\.\\d*\[02468\](?:\\.\\d+)*)${extract.suffix}"
+ 
```

Comment:
Remove this trailing newline.

Reply:
Done, this and tabs. Thank you for pointing out.